### PR TITLE
CDAP-3262 Errors in Windows Scripts

### PR DESCRIPTION
**Pull request by awholegunch**
Errors in Windows scripts: 
- If you change the path, you need to reset when you are done, as the change persists after the script completes. (It ends when the command window is closed.) 
- "GOTO" does not use a colon (unless you are calling the ":EOF"). "CALL" however does use a colon.

Trimmed trailing whitespace. Fixes files to all have Windows CRLF line endings.

### CODE REVIEW

File: cdap-examples/StreamConversion/bin/send-events.bat [hunk 1]
```diff
@@ -1,82 +1,84 @@
...
+REM ##
+REM ## http://www.apache.org/licenses/LICENSE-2.0
+REM ##
+REM ## Unless required by applicable law or agreed to in writing, software
+REM ## distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+REM ## WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+REM ## License for the specific language governing permissions and limitations under
+REM ## the License.
+REM ##
+REM #################################################################################
```
Comment (wolf31o2) [inline cdap-examples/StreamConversion/bin/send-events.bat]: :: ?

File: cdap-examples/UserProfiles/bin/add-users.bat [hunk 2]
```diff
@@ -1,91 +1,93 @@
...
+REM ##
+REM ## http://www.apache.org/licenses/LICENSE-2.0
+REM ##
+REM ## Unless required by applicable law or agreed to in writing, software
+REM ## distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+REM ## WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+REM ## License for the specific language governing permissions and limitations
+REM ## under the License.
+REM ##
+REM ############################################################################
```
Comment (wolf31o2) [inline cdap-examples/UserProfiles/bin/add-users.bat]: :: here, too

**Review by wolf31o2 — CHANGES_REQUESTED** [review]
Couple comments.

File: cdap-examples/StreamConversion/bin/send-events.bat (hunk 1 above)
Comment (awholegunch) [reply cdap-examples/StreamConversion/bin/send-events.bat]: Yes, we can make these identical.

File: cdap-examples/UserProfiles/bin/add-users.bat (hunk 2 above)
Comment (awholegunch) [reply cdap-examples/UserProfiles/bin/add-users.bat]: ibid.

**Review by wolf31o2 — APPROVED** [review]
👍 LGTM...

In the future, you should hold off on the line-ending changes until the content changes are made, then do the line-endings as part of your squash/rebase. Otherwise, it's nearly impossible to tell what's changed.

**Pull request merged**